### PR TITLE
dec ux update

### PR DESCRIPTION
**Pull request by Blackbaud-ToddRoberts**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-ux-update-201912](https://developer.blackbaud.com/skyux?_sp.spa=pr-ux-update-201912)

File: src/app/blog/index.html [hunk 1]
```diff
@@ -5,6 +5,23 @@
...
+			<stache-page-anchor>
+				Design update — December 2019
+			</stache-page-anchor>
+		</header>
+		<section>
+			<h3 class="sky-subsection-heading">In the works</h3>
+        <h4 class="sky-emphasized"><a href="https://github.com/blackbaud/skyux-list-builder/issues/22">List view switcher</a></h4>
+					<p>We are porting the list view switcher from SKY UX 1.</p>
+        <h4 class="sky-emphasized"><a href="https://github.com/blackbaud/skyux2/issues/1393">Inline delete confirmation</a></h4>
+					<p>We are adding an inline confirmation interaction for when users delete items from a list.</p>
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/blog/index.html]: I'd delete "for" and then call it good.

Comment (Blackbaud-ToddRoberts) [reply src/app/blog/index.html]: updated

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-ux-update-201912](https://developer.blackbaud.com/skyux?_sp.spa=pr-ux-update-201912)

**Pull request merged**